### PR TITLE
add xsimd@8.1.0

### PR DESCRIPTION
**Pull request by tongke6**
add new module xsimd and its first bcr version 8.1.0

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (xsimd) have been updated in this PR. Please review the changes.

**Comment by tongke6** [comment]
@bazel-io skip_check unstable_url

File: modules/xsimd/8.1.0/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
```
Comment (meteorcloudy) [inline modules/xsimd/8.1.0/presubmit.yml:10]: nit: If you want to also verify against the upcoming Bazel 8, you can add `8.0.0rc2` here. (8.x doesn't work yet)

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

**Review by meteorcloudy — APPROVED** [review]

File: modules/xsimd/8.1.0/presubmit.yml (hunk 1 above)
Comment (tongke6) [reply modules/xsimd/8.1.0/presubmit.yml:10]: Got it

**Pull request merged**